tools/postgres/schema.sql

```diff
@@ -56,7 +56,8 @@ CREATE TABLE webknossos.annotation_layers(
   _annotation CHAR(24) NOT NULL,
   tracingId CHAR(36) NOT NULL UNIQUE,
   typ webknossos.ANNOTATION_LAYER_TYPE NOT NULL,
-  name VARCHAR(256),
+  name VARCHAR(256) NOT NULL CHECK (name ~* '^[A-Za-z0-9\-_\.]+$'),
+  UNIQUE (name, _annotation),
```

Comment:
Should those constraints also get names, similar to the evolution? Or are the names only needed to allow the reverse evolution?

Reply:
postgres generates default names using a fixed schema, these are the ones I use in the reversion